### PR TITLE
EPRS: 7.2.1 release notes

### PR DESCRIPTION
**Pull request by ccestes**
## What Changed?

- Added new release notes topic
- Added release notes content
- Updated index file
- Updated supported platforms

### CODE REVIEW

File: product_docs/docs/eprs/7/01_introduction/04_supported_jdk_versions.mdx [hunk 1]
```diff
@@ -11,7 +11,7 @@ Replication Server is certified to work with the following Java platforms:
 | SLES 12                                 | Red Hat OpenJDK 8                                                                                               |
 | PPCLE RHEL 8                            | Red Hat OpenJDK 8 |
 | RHEL 7 and 8                            |  -   Red Hat OpenJDK 8     <br /> <br /> -   Oracle JDK 8     <br />         |
-| Windows 2012 R2, 2016, and 2019         | Red Hat OpenJDK 8                                                                                          |
+| Windows 2019                            | Red Hat OpenJDK 8                                                                                          |
```
Comment (drothery-edb) [inline product_docs/docs/eprs/7/01_introduction/04_supported_jdk_versions.mdx:14]:
```suggestion
| Windows 2016, 2019                            | Red Hat OpenJDK 8                                                                                          |
```

File: product_docs/docs/eprs/7/eprs_rel_notes/16_eprs_rel_notes_7.2.1.mdx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Version 7.2.1"
+---
+
+
+New features, enhancements, bug fixes, and other changes in Replication Server 7.2.1 include the following:
+
+| Type    | Description |
+| ------- |------------ |
+| Bug fix | Fixed the issue that caused the Publication creation failure due to a shadow table name conflict. #83414 |
```
Comment (mlewandowski-edb) [inline product_docs/docs/eprs/7/eprs_rel_notes/16_eprs_rel_notes_7.2.1.mdx:10]: It's my understanding that the support case number should be listed after the sentence using this convention:
"[Support Ticket #<case number]"

Please update this note to use this convention.

File: product_docs/docs/eprs/7/eprs_rel_notes/16_eprs_rel_notes_7.2.1.mdx [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+| Bug fix | Fixed the issue where the Publication database registration fails for a database that has intarray extension installed on it. #83759 |
```
Comment (mlewandowski-edb) [inline product_docs/docs/eprs/7/eprs_rel_notes/16_eprs_rel_notes_7.2.1.mdx:11]: It's my understanding that the support case number should be listed after the sentence using this convention:
"[Support Ticket #<case number]"

Please update this note to use this convention.

File: product_docs/docs/eprs/7/01_introduction/04_supported_jdk_versions.mdx [hunk 4]
```diff
@@ -11,7 +11,7 @@ Replication Server is certified to work with the following Java platforms:
 | SLES 12                                 | Red Hat OpenJDK 8                                                                                               |
 | PPCLE RHEL 8                            | Red Hat OpenJDK 8 |
 | RHEL 7 and 8                            |  -   Red Hat OpenJDK 8     <br /> <br /> -   Oracle JDK 8     <br />         |
-| Windows 2012 R2, 2016, and 2019         | Red Hat OpenJDK 8                                                                                          |
+| Windows 2016, 2019                            | Red Hat OpenJDK 8                                                                                          |
```
Comment (mlewandowski-edb) [inline product_docs/docs/eprs/7/01_introduction/04_supported_jdk_versions.mdx:14]: Based on a discussion with @kelpoole, when normal support for a platform ends, even if extended support for it is still available, EDB's normal policy is to discontinue support for our software on that platform.  As such, this change should be reverted back to the previous version where only Windows 2019 listed is listed.

In addition the "Windows 2019" listing should be changed to "Windows Server 2019".

**Review by mlewandowski-edb — CHANGES_REQUESTED** [review]
See comments on supported platforms and support case references in release notes.

**Review by mlewandowski-edb — APPROVED** [review]

**Review by mlewandowski-edb — APPROVED** [review]

**Pull request merged**